### PR TITLE
Spec - Wallet UI - IBC Transfer

### PR DESCRIPTION
**Pull request by jurevans**
Adding additional specs for IBC Transfer from the wallet UI.

### CODE REVIEW

File: src/architecture/clients/web-wallet/ibc.md [hunk 1]
```diff
@@ -2,30 +2,106 @@
...
+- A non-default `portId`, if necessary, though in most cases, the default of `transfer` would likely be used.
 
 ## Resources
 
 - [Anoma Ledger IBC Rust Docs](https://docs.anoma.network/master/rustdoc/anoma/ledger/ibc/)
 - [HackMD IBC Summary](https://hackmd.io/H2yGO3IQRLiWCPWwQQdVow)
 - [ibc-rs](https://github.com/informalsystems/ibc-rs/)
 - [ICS020 - Fungible Token Transfers](https://github.com/cosmos/ibc/blob/master/spec/app/ics-020-fungible-token-transfer/README.md)
-- [ICS027 - Interchain Accounts](https://github.com/cosmos/ibc/tree/master/spec/app/ics-027-interchain-accounts)
 - https://spec.anoma.network/master/architecture/namada/ibc.html
```
Comment (tzemanovic) [inline src/architecture/clients/web-wallet/ibc.md:100]:
```suggestion
- <https://specs.anoma.net/master/architecture/namada/ibc.html>
```

File: src/architecture/clients/web-wallet/ibc.md [hunk 2]
```diff
@@ -2,30 +2,106 @@
...
 - https://docs.cosmos.network/master/ibc/overview.html
```
Comment (tzemanovic) [inline src/architecture/clients/web-wallet/ibc.md:101]: this has been moved, not sure if this is the same page?
```suggestion
- <https://ibc.cosmos.network/main/ibc/overview.html>
```

File: src/architecture/clients/web-wallet/ibc.md [hunk 3]
```diff
@@ -2,30 +2,106 @@
...
 - https://ibcprotocol.org/
```
Comment (tzemanovic) [inline src/architecture/clients/web-wallet/ibc.md:102]: to render them as links in html
```suggestion
- <https://ibcprotocol.org/>
```

File: src/architecture/clients/web-wallet/ibc.md [hunk 4]
```diff
@@ -2,30 +2,106 @@
...
+
+Cosmos relayers:
+
+- https://hub.mintscan.io/ibc-network
+- https://www.mintscan.io/osmosis/relayers
```
Comment (tzemanovic) [inline src/architecture/clients/web-wallet/ibc.md:107]:
```suggestion
- <https://hub.mintscan.io/ibc-network>
- <https://www.mintscan.io/osmosis/relayers>
```

File: src/architecture/clients/web-wallet/ibc.md (hunk 2 above)
Comment (jurevans) [reply src/architecture/clients/web-wallet/ibc.md:101]: It's the same page (recently moved). Thanks!

**Review by tzemanovic — APPROVED** [review]
LGTM, thanks!

**Review by cwgoes — APPROVED** [review]

**Pull request merged**